ocs_ci/ocs/cluster.py

```diff
@@ -2972,3 +2973,53 @@ def get_lvm_full_version():
     image = getattr(redhat_operators_catalogesource_ocs, "data")["spec"]["image"]
     full_version = image.split(":")[1]
     return full_version
+
+
+def get_mds_standby_replay_info():
```

Comment:
Please follow Google Style of Python Docstrings

Reply:
Done